assignment5.txt

```diff
@@ -0,0 +1,18 @@
+Yael Kelmer.
+
+1. This is how you would call the function inside of the main() function:
+
+int main () {
+
+	int a = 1; /* this initializes a value for a */
+	float b = 2.5; /* this initializes a value for b */
+	int c = 4; /* this initializes a value for c */
+
```

Comment:
right! but make sure that the function is declared before the main function to avoid a warning.